runtime/js/jslib.js

```diff
@@ -61,13 +61,14 @@ function caml_trampoline(res) {
 }
 
 //Provides:caml_trampoline_return
-function caml_trampoline_return(f, args) {
-  return { joo_tramp: f, joo_args: args };
+function caml_trampoline_return(f, args, direct) {
+  return { joo_tramp: f, joo_args: args, joo_direct: direct };
 }
 
 //Provides:caml_stack_depth
 //If: effects
-var caml_stack_depth = 0;
+var caml_stack_depth = 10; // Initialized to a non-zero value in case of
```

Comment:
Can you say more about this ? why 10 and not just 1 ?

Reply:
I believe this change is no longer relevant and can be reverted.